### DISCUSSION TITLE
Auto add network to Metamask did not kick in

### DISCUSSION THREAD

**Discussion by yasiryagi** [subquery/subql#966] [Frontier TestNet - Bug Reporting]
When initially using the indexer UI The page got stuck at Connect wallet to use Indexer App

**Comment by corvin2473** [comment]
I had the same problem
![metamask_auto_add_fail](https://user-images.githubusercontent.com/80776583/170081956-d82ed1ff-1a23-47ba-a27d-d0f24419f643.jpg)
.
